jobs/terraform-deploy/terraform-deploy.groovy

```diff
@@ -0,0 +1,61 @@
+job {
+  name 'terraform-deploy'
+  description('Deploy and automatically update Terraform environments upon repository changes')
+  parameters {
+    choiceParam("action", ["apply", "destroy"], "Select if you want to apply changes to environments or destroy them.")
```

Comment:
Either one of those is fine, but I think you could just move the message to the next line and it would fit in, i.e.:

```
   choiceParam("action", ["apply", "destroy"], 
               "Select if you want to apply changes to environments or destroy them.")
```